Tests/PointFreeTests/PricingTests.swift

```diff
@@ -51,22 +52,31 @@ class PricingTests: TestCase {
     #endif
   }
 
-  func testPricingLoggedIn() {
-    let conn = connection(from: authedRequest(to: .pricing(nil, nil)))
-    let result = conn |> siteMiddleware
+  func testPricingLoggedIn_NonSubscriber() {
+    AppEnvironment.with(\.stripe.fetchSubscription .~ const(throwE(unit))) {
```

Comment:
changed this test to explicitly state that a user is logged in and they are not a subscriber.